### PR TITLE
PCI Enable Master Mode Fix

### PR DESCRIPTION
**Pull request by slacrherbst**

### CODE REVIEW

File: common/driver/axis_gen2.c [hunk 1]
```diff
@@ -536,6 +536,7 @@ void AxisG2_SeqShow(struct seq_file *s, struct DmaDevice *dev) {
    seq_printf(s,"     Hw Read Buff Count : %i\n",hwData->hwRdBuffCnt);
    seq_printf(s,"           Cache Config : 0x%x\n",(ioread32(&(reg->cacheConfig))));
    seq_printf(s,"            Desc 128 En : %i\n",hwData->desc128En);
+   seq_printf(s,"            Envable Var : 0x%x\n",(ioread32(&(reg->enableVer))));
```
Comment (ruck314) [inline common/driver/axis_gen2.c]: Is it possible for the enableVer to be 0x0 if the driver is loaded?

Comment (slacrherbst) [reply common/driver/axis_gen2.c]: Yes if it is a older version of the firmware.

Comment (slacrherbst) [reply common/driver/axis_gen2.c]: But no the driver should enable it.

Comment (slacrherbst) [reply common/driver/axis_gen2.c]: But that register also has other useful information.

**Pull request merged**